app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,20 @@
 <resources>
     <string name="app_name">kotlinOne</string>
+    <string name="activityMainTextView1">Hello!!!!</string>
+    <string name="MovieDetailsTextViewName">Avengers: \nEnd Game</string>
+    <string name="path_back">Back</string>
+    <string name="textViewTegLine">Action, Adventure, Fantasy</string>
+    <string name="MovieDetailsTextViewSomeId">13+</string>
+
+    <string name="textViewReview">125 Reviews</string>
+    <string name="textViewStoryLine">Storyline</string>
```

Comment:
@kamar19 Please decide which type of naming to use and use only one. You use Camel case and Snake case for strings at the same time. It is not a very good approach.

Reply:
Accepted. Correcting.